src/styles/index.scss

```diff
@@ -2,2 +1,137 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__model {
+    margin-top: 40px;
+    width: 166px;
+    height: 36px;
```

Comment:
Fixed width and height on text containers can lead to text overflow or clipping issues. Use padding along with min-height or min-width as needed, and ensure text has enough room to scale with different screen sizes or font settings.